src/modules/video_player/index.js

```diff
@@ -94,6 +122,14 @@ class VideoPlayerModule {
             $(VIDEO_PLAYER_SELECTOR).on('click', '.player-overlay.player-fullscreen-overlay', handlePlayerClick);
         }
     }
+
+    muteInvisibleTabs() {
+        $(document).off(visibilityChange, handleVisibilityChange);
```

Comment:
this would be a lot more trivial to just use jQuery's blur and focus events..

```
$(window).on('blur', () => console.log('the page is not active'));
$(window).on('focus', () => console.log('the page is active'));
```